plugins/woocommerce/changelog/update-disable-reports-setting

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Disable turning the current Analytics feature off.
```

Comment:
Maybe it's worth mentioning that this feature option will be hidden for fresh installs?